test/test_tensor_creation_ops.py

```diff
@@ -1960,6 +1960,99 @@ def test_inference(default_dtype):
         test_inference(torch.float64)
         test_inference(torch.float32)
 
+    @suppress_warnings
+    @onlyCPU
+    def test_tensor_factory_ndarrays(self, device):
+        """
+        Tests torch.tensor for multiple numpy arrays that are embedded (recursively) within list(s). 
+        """
+
+        repeats = 10
+        np_types = list(torch_to_numpy_dtype_dict.values())
```

Comment:
Use the @dtypes decorator with the corresponding torch dtypes and map them them to numpy dtypes using this dict in the test. That will instantiate a test for each dtype, which improves test granularity.

Reply:
Done.